test/parse_test.js

```diff
@@ -33,6 +33,12 @@ describe('parse()', function() {
     expect(JSON.stringify(result.json())).to.equal(outputJSON);
   });
   
+  it('should parse AsyncAPI document passed as JS object', async function() {
+    const object = JSON.parse(inputJSON);
+    const result = await parser.parse(object, { path: __filename });
+    expect(JSON.stringify(result.json())).to.equal(outputJSON);
```

Comment:
Most likely parsed `inputJSON` is not the same as `outputJSON`. Probably you should to add a new `outputJSON` related to `inputJSON`. And try to test it locally too after pushing new changes :)

Reply:
I have created a  new `outputJSONForObjectInput` for expected output related to `inputJSON` and all the tests have passed 😄. Thank you for your help, please have a look. 
<img width="595" alt="parse_test" src="https://user-images.githubusercontent.com/66796715/120537413-e4a61380-c402-11eb-919c-42a31d446d66.PNG">

Reply:
Also let me know if I should remove the changes in package.json and package-lock.json. They were added because I installed eslint latest version as test and lint scripts were not running on my system for some reason maybe a versin mismatch.

Reply:
@vishesh13byte Eslint is a dev dependency so it's ok to have latest version. It can stay.